src/torchmetrics/functional/__init__.py

```diff
@@ -43,6 +43,8 @@
 from torchmetrics.functional.image.tv import total_variation
 from torchmetrics.functional.image.uqi import universal_image_quality_index
 from torchmetrics.functional.nominal.cramers import cramers_v
+from torchmetrics.functional.nominal.pearson import pearsons_contingency_coefficient
+from torchmetrics.functional.nominal.tschuprows import tschuprows_t
```

Comment:
is there a reason we are not importing the `_matrix` versions?

Reply:
Actually, no reason not to add here. I'll add that :]